README.md

```diff
@@ -15,3 +15,61 @@ See `main.rb` and `Rakefile` for the usage of the action api and rake task.
 
 1. To start the server, run `rake thin:start`
 2. To restart the server, run `rake thin:restart`
+
+# Using Docker
+## Run Rake Task for Log Parsing
+Log parser sends logs to Elasticsearch for analytics. There are three ways to run log parser rake task.
+1. Send logs to an Elasticsearch cluster running on localhost.
+2. Send logs to a single node Elasticsearch cluster running on Docker.
+3. Send logs to a three node Elasticsearch cluster running on Docker.
+
+### Local ES
+- Start:
+`docker-compose up -d`
+- Exec to Container:
+`docker-compose exec grunt sh`
+- Stop:
+`docker-compose down`
+### Single Node ES
+- Start:
+`docker-compose -f docker-compose.es.yml up -d`
+- Exec to Container:
+`docker-compose -f docker-compose.es.yml exec grunt sh`
+- Stop:
+`docker-compose -f docker-compose.es.yml down`
+### Three Node ES
+- Start:
+`docker-compose -f docker-compose.escluster.yml up -d`
+- Exec to Container:
+`docker-compose -f docker-compose.escluster.yml exec grunt sh`
+- Stop:
+`docker-compose -f docker-compose.escluster.yml down`
+
+## Settings
+In `config/settings.yml` use the following settings for connecting to Elasticsearch when running from Docker.
+### Using Local ES
+```
+elasticsearch:
+  host: host.docker.internal
+  port: 9200
+```
+
+### Using Docker ES
+```
+elasticsearch:
+  host: es
+  port: 9200
+```
+
+## Command for Rake Task
+Download logs from S3 using the script in `fetchs3logs.sh`.
+Set the values for `BUCKET`, `LOG_TYPE` and `LOG_FILE_PREFIX`.
+Put log files in a folder inside this repository.
```

Comment:
@mfarix add this
`The tmp directory inside this repo might be a good candidate here`